test/e2e_node/runtime_conformance_test.go

```diff
@@ -51,7 +51,7 @@ var _ = framework.KubeDescribe("Container Runtime Conformance Test", func() {
 				Testname: Container Runtime, restart policy
 				Description:  If the restart policy is set to ‘Always’, Pod MUST be started when terminated, If restart policy is ‘OnFailure’, Pod MUST be started only if it is terminated with non-zero exit code. If the restart policy is ‘Never’, Pod MUST never be restarted. All these three test cases MUST verify the restart counts accordingly.
 			*/
-			framework.ConformanceIt("it should run with the expected status [NodeConformance]", func() {
+			It("it should run with the expected status [NodeConformance]", func() {
```

Comment:
this file should be ported.

Reply:
#67103 (this seems less obvious to me, but let's see how far we can gert)

Reply:
Container restart behavior is a portable pod feature, so it should be covered by conformance.